src/rustup-cli/rustup_mode.rs

```diff
@@ -219,14 +219,33 @@ pub fn cli() -> App<'static, 'static> {
                 .arg(Arg::with_name("toolchain")
                      .required(true)))
             .subcommand(SubCommand::with_name("unset")
-                .about("Remove the override toolchain for a directory"))
+                .about("Remove the override toolchain for a directory")
+                .after_help("If \"--path\" argument is present, removes the override toolchain for \
+                specified directory. If \"--nonexistent\" argument is present, removes the override \
+                toolchain for all nonexistent directories. Otherwise, removes the override toolchain \
+                for current directory.")
```

Comment:
Please move this text to help.rs.

"toolchain for specified directory" -> "toolchain for the specified directory"

"toolchain for current directory" -> "toolchain for the current directory"